doc/source/math/trafos/fourier_transform.rst

```diff
@@ -98,7 +98,7 @@ Discretization of the Fourier transform operator means evaluating the Fourier in
     :label: discr_function
 
 with coefficients :math:`\bar f = (f_0, \dots, f_{n-1}) \in \mathbb{C}^n` and functions
-:math:`\phi_0, \dots, \phi_{n-1}`. This approach follows from the way , but can be
+:math:`\phi_0, \dots, \phi_{n-1}`.
```

Comment:
I don't know if this really belongs in this PR, but I a, fine with it if it doesn't bother you.

Reply:
Yeah I looked up this doc page and found the error. Not strictly part of the PR, but very closely related :wink: